wiki/quest-modding.md

```diff
@@ -1,14 +1,58 @@
+* [BeatSaberQuestInstaller (windows only GUI for installing BMBF)](https://github.com/NyanBlade/BeatSaberQuestInstaller/releases)
+* [BMBF installed via SideQuest](https://github.com/kihecido/BMBF/releases)
+
+For new users, BeatSaberQuestInstaller may be easier to use as it handles some of the more confusing parts of modding/updating a modded game, such as restoring scores and installing adb.
+
+### Installing with BeatSaberQuestInstaller
+
+Before starting the process go to the utilities tab and backup your scores, if you've previously modded Beat Saber you also need to go there to uninstall the game, make sure you backup **BEFORE** uninstalling.
+
+![BeatSaberQuestInstaller](./images/beginners-guide/bsqioptions.png)
+
+Once you've done what you need, if you're a new user follow all the steps from step 1 and if you're a returning user skip to step 4 in the installer, after completing the steps open BMBF from the channels tab in Oculus TV or Unknown Sources in your library.
+
+### Installing BMBF with SideQuest
+
+If you haven't already, download and setup [SideQuest](https://sidequestvr.com/#/setup-howto)
+
+Open SideQuest and connect your Quest to your PC. If you've previously modded Beat Saber or have scores you want to backup, go to `My Apps` located in the top bar of the window and find Beat Saber.
+
+Click the cog next to it then press the `BACKUP GAME DATA` button, if you have a modded game you also need to uninstall it by pressing the `UNINSTALL APP` button.
+
+![SideQuestUninstall](./images/beginners-guide/squninstall.png)
+
+You can later restore your save from the same menu, after modding.
+
+Install the latest BMBF apks via the `Install APK from folder` button shown below.
+
+![InstallAPK](./images/beginners-guide/apkfromfolder.png)
+
+Once they've successfully installed, make sure you have the latest version of Beat Saber installed and unmodded **RUN IT ONCE BEFORE MODDING** then open BMBF from Oculus TV channels tab or Unknown Sources in your library, follow each step exactly as you're told and
+once you have you should be put into [bsaber.com](https://www.bsaber.com) this is where you can download any custom songs available.
+
+## Restoring save data
+
+### Restoring from BeatSaberQuestInstaller
+
+Make sure you've ran the game modded once and go to the `UTILITIES` tab in the app and press the `TRANSFER SCORES` button.
+
+### Restoring from SideQuest
+
+Go back to the same menu you used to backup your data and click the `OPEN BACKUPS` button.
+From there, find your newest backup folder, open it, and delete the Mods folder (this is to prevent potentially old or incompatible mods from breaking your game).
+
+Afterwards, go back to the menu and press the circular arrows located beside your latest backup; this is to restore your scores.
+The button is highlighted in the image [here](https://github.com/beat-saber-modding-group/wiki/blob/master/wiki/quest-modding.md#installing-bmbf-with-sidequest).
```

Comment:
Instead of linking to the github page for this header, you can link to just the header name, `#installing-bmbf-with-sidequest`, and when it is clicked the browser will scroll to that section on the current page.